code/modules/mob/dead/observer/observer.dm

```diff
@@ -598,6 +598,71 @@
 				return
 			SSticker.mode.spawn_larva(src, mother)
 
+/mob/dead/verb/join_as_necromorph()
+	set category = "Ghost"
+	set name = "Join as Necromorph"
+
+	var/mob/N = src
+
+	if(SSticker.current_state < GAME_STATE_PLAYING)
+		to_chat(usr, "<span class='warning'>The game hasn't started yet!</span>")
+		return
+
+	if(jobban_isbanned(usr, "Alien"))
+		to_chat(usr, "<span class='warning'>You are banned from playing aliens and cannot spawn as a Necromorph.</span>")
+		return
+
+	var/list/necromorph_list = list()
+
+	for(var/mob/living/carbon/human/species/zombie/necromorph/A in GLOB.alive_mob_list)
+		if(istype(A) && !A.client)
```

Comment:
```suggestion
		if(!A.client)
```
No need to check for the type, as you are already doing that in the `for` lookup.

Reply:
Though ideally there's a global var of necromorphs instead. Same as for xenos or humans.